.eslintrc

```diff
@@ -0,0 +1,509 @@
+
```

Comment:
This is just the ESLint file I copied from `web-apps`. Didn't put the script runner in place, but figured this could be useful.